core/domain/base_model_validators_test.py

```diff
@@ -152,15 +153,19 @@ def setUp(self):
     def test_error_is_raised_if_fetch_external_properties_is_undefined(self):
         with self.assertRaisesRegexp(
             NotImplementedError,
-            r'The _get_external_id_relationships\(\) method is missing from the'
-            ' derived class. It should be implemented in the derived class.'):
+            re.escape(
+                'The _get_external_id_relationships() method is missing'
+                ' from the derived class. It should be implemented in the'
+                ' derived class.')):
             MockBaseModelValidator().validate(self.invalid_model)
 
     def test_error_is_get_external_model_properties_is_undefined(self):
         with self.assertRaisesRegexp(
             NotImplementedError,
-            r'The _get_external_model_properties\(\) method is missing from the'
-            ' derived class. It should be implemented in the derived class.'):
+            re.escape(
+                'The _get_external_model_properties() method is missing'
+                ' from the derived class. It should be implemented in the'
+                ' derived class.')):
```

Comment:
While we are here, could we put the spaces at the ends of the lines?
```suggestion
                'The _get_external_model_properties() method is missing '
                'from the derived class. It should be implemented in the '
                'derived class.')):
```

Reply:
Ditto elsewhere.

Reply:
Done.